tools/tracegen/TraceHeaderWriter.cpp

```diff
@@ -214,8 +214,9 @@ TraceHeaderWriter::tpTemplate(FILE *fd, unsigned int overhead, unsigned int test
 	char *testNop =  NULL;
 	char *testMacroTemplate = (char *)  "#define TrcEnabled_%s  (%s_UtActive[%u] != 0)\n";
 	char *testNopTemplate = (char *) "#define TrcEnabled_%s  (0)\n";
+	size_t parmBufLen = (parmCount * sizeof(char) * 5) + 1;
```

Comment:
My understanding is that the language guarantees that `sizeof(char) == 1`. I'm not sure the point of multiplying by `sizeof(char)`, or is it just for clarity?

Reply:
It is taken from the original code:
```
parmString = (char *)Port::omrmem_calloc(1, (parmCount * sizeof(char) * 5) + 1);
```

I removed `* sizeof(char)`.